lighthouse-core/computed/metrics/total-blocking-time.js

```diff
@@ -39,30 +38,29 @@ class CumulativeLongQueuingDelay extends ComputedMetric {
    * @param {number} interactiveTimeMs
    * @return {number}
    */
-  static calculateSumOfLongQueuingDelay(topLevelEvents, fcpTimeInMs, interactiveTimeMs) {
+  static calculateSumOfBlockingTime(topLevelEvents, fcpTimeInMs, interactiveTimeMs) {
     if (interactiveTimeMs <= fcpTimeInMs) return 0;
 
-    const threshold = CumulativeLongQueuingDelay.LONG_QUEUING_DELAY_THRESHOLD;
-    const longQueuingDelayRegions = [];
-    // First identifying the long queuing delay regions.
+    const threshold = TotalBlockingTime.BLOCKING_TIME_THRESHOLD;
+    const blockingRegions = [];
     for (const event of topLevelEvents) {
-      // If the task is less than the delay threshold, it contains no Long Queuing Delay Region.
+      // If the task is less than the delay threshold, it contains no Blocking Region.
       if (event.duration < threshold) continue;
-      // Otherwise, the duration of the task before the delay-threshold-sized interval at the end is
-      // considered Long Queuing Delay Region. Example assuming the threshold is 50ms:
+      // Otherwise, the duration of the task beyond blocking time threshold at the beginning is
+      // considered Blocking Region. Example assuming the threshold is 50ms:
       //   [              250ms Task                   ]
-      //   |  Long Queuing Delay Region  |   Last 50ms |
-      //               200 ms
-      longQueuingDelayRegions.push({
-        start: event.start,
-        end: event.end - threshold,
+      //   | First 50ms |     Blocking Time Region     |
+      //                            200 ms
+      blockingRegions.push({
+        start: event.start + threshold,
+        end: event.end,
         duration: event.duration - threshold,
       });
     }
 
-    let sumLongQueuingDelay = 0;
-    for (const region of longQueuingDelayRegions) {
-      // We only want to add up the Long Queuing Delay regions that fall between FCP and TTI.
+    let sumBlockingTime = 0;
+    for (const region of blockingRegions) {
+      // We only want to add up the Blocking Time Regions that fall between FCP and TTI.
```

Comment:
not sure if it matters but is there significance to `Blocking Time Region` vs. `Blocking Region` or are they interchangeable?